### DISCUSSION TITLE
希望5.x 支持 企业微信新接口：自建应用与第三方应用对接

### DISCUSSION THREAD

**Discussion by zio-zhang** [w7corp/easywechat#2561] [Ideas]
企业微信最近做了很多安全升级，增加了部分接口
[自建应用与第三方应用的对接](https://developer.work.weixin.qq.com/document/path/95884)
1、userid转换：将代开发应用或第三方应用获取的密文open_userid转换为明文userid。
2、external_userid转换：将代开发应用或第三方应用获取的externaluserid转换成自建应用的externaluserid。

   
希望5.x能支持一下

**Comment by ljyljy0211** [comment]
不是有么
